.github/actions/build-publish-chainlink/action.yml

```diff
@@ -127,3 +141,22 @@ runs:
       build-args: |
         CHAINLINK_USER:chainlink
         ${{ env.shared-build-args }}
+
+  - if: inputs.sign-images == 'true'
+    name: Install cosign
+    uses: sigstore/cosign-installer@1e95c1de343b5b0c23352d6417ee3e48d5bcd422 # v1.4.0
+    with:
+      cosign-release: 'v1.4.0'
+
+  - if: inputs.sign-images == 'true'
+    name: Write signing key to disk (only needed for `cosign sign --key`)
+    shell: sh
+    run: echo "${{ inputs.cosign-private-key }}" > cosign.key
```

Comment:
Can we not pass this in stdin to `cosign sign`?

Reply:
Unfortunately, `cosign` only reads the key from a file